File: web/src/app/core/models/detalle-venta.ts
import { Producto } from "./producto";
import { Venta } from "./venta";

export class DetalleVenta {
  constructor(
    public idDetalleVenta: number = null,
    public venta: Venta = null,
    public producto: Producto = null,
    public cantidad: number = null,
    public precioVenta: number = null
  ) {}
}

File: src/main/kotlin/com/bykenyodarz/sistemasventasweb/domain/model/DetalleVenta.kt
package com.bykenyodarz.sistemasventasweb.domain.model

data class DetalleVenta(
    var idDetalleVenta: Int? = null,
    var venta: Venta? = null,
    var producto: Producto? = null,
    var cantidad: Int? = null,
    var precioVenta: Double? = null
)
